Sources/StreamChat/APIClient/Endpoints/Payloads/UserPayloads.swift

```diff
@@ -26,6 +26,8 @@ enum UserPayloadsCodingKeys: String, CodingKey, CaseIterable {
     case devices
     case unreadCount = "unread_count"
     case language
+    case blockedUsers = "blocked_users"
```

Comment:
Unused?

Reply:
It's used here: https://github.com/GetStream/stream-chat-swift/blob/feature/block/Sources/StreamChat/APIClient/Endpoints/Payloads/CurrentUserPayloads.swift#L74